archive/generic-core-terms.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Security is core to our values, and we value the input of hackers acting in good-faith to help us maintain a high standard for the security and privacy for our users. This includes encouraging responsible vulnerability research and disclosure. This policy sets out our definition of good-faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return.
+
+# Expectations
+
+When working with us according to this policy, you can expect us to:
+
+- Work with you to understand and validate your report, including a timely initial response to the submission;
+- Work to remediate discovered vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to report a unique vulnerability, and your report triggers a code or configuration change.
+
+# Scope
+
+# Out-of-Scope
+
```

Comment:
Suggested insertion:

Assets or other equipment not owned by parties participating in this policy or not listed in "Systems In Scope."